ForMiniNTuple/batch/BatchGenerator.py

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/env python
```

Comment:
In the long run, we're going to want to remove or synthesize ForMiniMTuple with the rest of the code in a better way: ForMiniNTuple/batch/BatchGenerator.py is the kind of duplication we should try to avoid, for example.  For now it's OK, I guess.